### PR TITLE
First communication between Clorwell and Server Game

### PR DESCRIPTION
**Pull request by massix**
Theoretically it should fail the tests as I'm missing a patch on the protobuffers.
But still it's nice if it launches the compilation and everything.

### CODE REVIEW

**Comment by massix** [comment]
Clorwell is now communicating with the server.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,56 @@
...
+$ apt-get install libzmq3-dev
+$ git clone http://github.com/zeromq/jzmq.git jzmq-build
+$ cd jzmq-build
+$ ./autogen.sh
+$ ./configure --prefix=/opt/jzmq
+$ make
+$ make install
+```
+
+After that, we need to patch the protobuf definitions in order to have the Reflection
```
Comment (dchilot) [inline README.md:18]: We had migrated server-game to lite for the iOS client. Have you checked the interoperability on both side ?

**Comment by massix** [comment]
The choice of the runtime doesn't impact at all the way the messages are serialised: using the LITE runtime will compile the protobuf messages against a lighter version which is not using the reflection system (rtti) to determine the kind of the message, whereas the "heavy" one uses it.

Now, the way the Clojure bindings have been written relies on the Reflection to create the definition, hence I need to use the "heavy" runtime.

I've checked against the server, and the messages are exchanged correctly.

**Comment by dchilot** [comment]
In that case maybe it was a bit premature to migrate all to the lite version in the first place. I do not like the idea of patching the files from the repository (and it adds the risk of committing by mistake). Maybe we could find a solution that would allow to choose the version between lite or not without hacking the content ? (maybe with special proto files for the lite that would import the standard version)

**Pull request merged**